doc/user/quick-sql-grammar.md

```diff
@@ -377,15 +395,20 @@ When enabling audit columns use this setting to override default audit column na
 
 Show all settings, not just settings that are different from the default.
 
+## Document
+
+The database defined via QuickSQL is populated with the data generated by ![chancejs](https://github.com/chancejs/chancejs). If QSQL code has been generated from json document, then the document is kept under the `#document` section, and is used to populate the database with genuine data. See the ![Car Racing Example](../../test/DV/car_racing/1.qsql).
```

Comment:
- [ ] Change this to an 80 character rule
- [ ] Change QuickSQL and QSQL to Quick SQL